client/reducers/appReducer.js

```diff
@@ -6,6 +6,7 @@ export const initialState = {
   artsyURL: sd.ARTSY_URL,
   channel: sd.CURRENT_CHANNEL,
   forceURL: sd.FORCE_URL,
+  hasFeatures: sd.CURRENT_CHANNEL.type !== 'partner',
```

Comment:
Oh no another `features` 😛 . Should we call this `notPartnerChannel`?

Reply:
We use the 'hasFeatures' flag in many components, mind if I make a new pr that renames and replaces all instances of it, instead of changing here?

Reply:
Actually it doesn't affect other components when used from store.  Will update now but still add a follow-up pr.